collect_app/src/main/res/values/typography.xml

```diff
@@ -32,6 +32,10 @@
         <item name="android:textColor">@color/color_on_surface_medium_emphasis</item>
     </style>
 
+    <style name="TextAppearance.QuestionTitle" parent="TextAppearance.Material3.BodyLarge">
```

Comment:
How about using `?textAppearanceTitleLarge` instead of adding a new typography token? If that's too big, we could customise it so that it's 20sp.

Reply:
If 22sp is ok then yeah we can use `?textAppearanceTitleLarge` if not then customizing it would basically mean the same as I have done here right so why customizing `TitleLarge` would be better than customizing `BodyLarge`?

Below there is a table the first row contains the version with 22sp and the second one 20sp what do you think @seadowg @alyblenkin?
 
|  |  |
| ------------- | ------------- |
| ![Screenshot_1704833048](https://github.com/getodk/collect/assets/3276264/c7f1d6e7-30ff-4e6b-bf83-2f4ee8ac436b) | ![Screenshot_1704833050](https://github.com/getodk/collect/assets/3276264/56914953-0be9-4c56-be92-75622bf13463) |
| ![Screenshot_1704833096](https://github.com/getodk/collect/assets/3276264/6731ee7f-09c7-4a86-b057-d9df6b7a4fc9) | ![Screenshot_1704833100](https://github.com/getodk/collect/assets/3276264/a9bd61fa-2dbb-462c-be8a-afe9544a1360) |

Reply:
I think 20sp works better.

Reply:
> If 22sp is ok then yeah we can use ?textAppearanceTitleLarge if not then customizing it would basically mean the same as I have done here right so why customizing TitleLarge would be better than customizing BodyLarge?

Yeah sorry what I said was pretty unclear. I mean instead of introducing a new style (`QuestionTitle`), we just modify `?textAppearanceTitleLarge` for our theme:

**theme.xml**
```
<item name="textAppearanceTitleLarge">@style/TextAppearance.Collect.TitleLarge</item>
```

**typography.xml**
```
<style name="TextAppearance.Collect.TitleLarge" parent="TextAppearance.Material3.TitleLarge">
    <item name="android:textSize">20sp</item>
</style>
```

Reply:
Doing things like that with colors for example is ok like we do with:
```
    <style name="TextAppearance.Collect.Headline4" parent="TextAppearance.MaterialComponents.Headline4">
        <item name="android:textColor">@color/color_on_surface_high_emphasis</item>
    </style>
```

but customizing font size and still using the original style name seems misleading to me. There are actually two reasons why I'm not sure about that:
- `textAppearanceTitleLarge` is already used in at least one place (`crash_layout`). Doing what you described would affect that screen. 
- when I look for a good style for a new text I add check https://m3.material.io/styles/typography/type-scale-tokens. In such a case when you look at `TitleLarge` and its 22sp font size you may think that this is what you need. Then you apply this style and surprise something is wrong and you have to analyze what exactly. 

Do you think such font size changing for already existing styles is ok?

Reply:
> Do you think such font size changing for already existing styles is ok?

Good question! Yeah I do. Doing so is referenced in both the [Material 3 docs](https://m3.material.io/styles/typography/type-scale-tokens) and the [Material Components docs](https://github.com/material-components/material-components-android/blob/master/docs/theming/Typography.md#customization). My understanding is that from Material 2 onwards, the philosophy is that instead of adding your own components and styles on top of the Material styles, you're able to "theme" Material so that the standard Material components fit into our "brand". If we change the `?textAppearanceTitleLarge` it will also affect any Material components that might use that text appearance, but that's what we want.

> textAppearanceTitleLarge is already used in at least one place (crash_layout). Doing what you described would affect that screen.

It might be that we want to use a different token there (like a Headline one), or maybe 20sp is also good for that screen!

> when I look for a good style for a new text I add, then I check https://m3.material.io/styles/typography/type-scale-tokens. In such a case when you look at TitleLarge and its 22sp font size you may think that this is what you need. Then you apply this style and surprise something is wrong and you have to analyze what exactly.

That's a good point. When we customise these, we do lose some level of documentation. I think that's ok generally though as we should more the follow the ["roles"](https://m3.material.io/styles/typography/applying-type) when choosing a text appearance for something. I chose "Title Large" here as it seems to me to be the best description of the text. I could imagine that using (and maybe customising) a "Body" token would make more sense though given that question text can be of any length.

Reply:
Ok but if I have a choice between customizing `BodyLarge` and `TitleLarge` I think it's better to use the latter. There are two reasons:
- `TitleLarge` is used only in one place but `BodyLarge` in many
- `TitleLarge's` original font size is 22sp so the change is not that big as it would be in the case of `BodyLarge` (original front size 16sp)

Reply:
Yeah that makes sense to me. It does feel more like a "title" than "body" text as well.